war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -127,7 +128,7 @@ public Organization readOrganization(final String organizationCode) {
     *       organization data
     * @return id of the organization
     */
-   public String createOrganization(final Organization organization) {
+   public String createOrganization(final Organization organization) throws InvalidDocumentKeyException {
       String id = dataStorage.createDocument(LumeerConst.Organization.COLLECTION_NAME, organization.toDataDocument());
       databaseInitializer.onOrganizationCreated(id);
       securityFacade.addOrganizationUserRole(organization.getCode(), userFacade.getUserEmail(), LumeerConst.Security.ROLE_MANAGE);
```

Comment:
Move this line to `databaseInitilizer.onOrganizationCreated` method and don't throw exception because we know, that we add correct role.